lib/scenic/scene.ex

```diff
@@ -195,10 +195,10 @@ defmodule Scenic.Scene do
   to the cursor input, the text field captures text input (and
   maybe transforms its graph to show that it is selected).
 
-  Captured input types are should be released when no longer
+  Captured input types should be released when no longer
   needed so that normal operation can resume.
 
-  The input messages re not passed on to other scene if
+  The input messages are passed on to other scene if
```

Comment:
I was unsure of the original intent here, if the input messages are passed to other scenes or not.
Also is it like they are passed up the tree to other scenes?